EatFit.Web/Web.config

```diff
@@ -98,27 +106,28 @@
       <add name="ScriptResource.axd_GET,HEAD" path="ScriptResource.axd" verb="GET,HEAD" type="System.Web.Handlers.ScriptResourceHandler, System.Web.Extensions, Version=1.0.61025.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" preCondition="integratedMode,runtimeVersionv2.0"/>
       <add name="*.asmx_*" path="*.asmx" verb="*" type="System.Web.Script.Services.ScriptHandlerFactory, System.Web.Extensions, Version=1.0.61025.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" preCondition="integratedMode,runtimeVersionv2.0"/>
     </handlers>
+
     <validation validateIntegratedModeConfiguration="false"/>
-    <defaultDocument>
-      <files>
-        <add value="Default.html"/>
-      </files>
-    </defaultDocument>
     <httpRedirect enabled="false" destination="http://eatfit.ucdavis.edu" exactDestination="false"/>
-    <httpWarmup>
+    <!--<httpWarmup>
       <requests>
         <add requestUrl="/" allowedResponseCodes="200-399" warmupContext="" sendMode="Asynchronous"/>
       </requests>
-    </httpWarmup>
+    </httpWarmup>-->
+    <!--<defaultDocument>
+      <files>
+        <add value="Default.html" />
+      </files>
+    </defaultDocument>-->
   </system.webServer>
+
   <!-- This section allows a temporary password to be sent to a user's registered e-mail
      address as a result of a password reset request. -->
   <system.net>
     <mailSettings>
       <smtp from="eatfit@ucdavis.edu">
-        <network host="smtp.ucdavis.edu" password="" userName=""/>
+        <network host="smtp.sparkpostmail.com" password="a6876b1b-3d72-42ce-a77b-0b8f9c454ff2" userName=""/>
```

Comment:
Checking this in would let anyone spam from our account. Can you get it in app settings or config secrets somehow?